src/pages/api/academic-grants.ts

```diff
@@ -104,7 +105,10 @@ async function handler(req: NextApiRequest, res: NextApiResponse): Promise<void>
         // Document upload
         conn.sobject('ContentVersion').create(
           {
-            Title: `[PROPOSAL] ${application.Project_Name__c} - ${createdLeadID}`,
+            Title: `[PROPOSAL] ${truncateString(
+              application.Project_Name__c || '',
```

Comment:
if `application.Project_Name__c` is empty (not file uploaded), we get `undefined` as value here? IMO this should be considered on the `truncateString` util and just use `truncateString(application.Project_Name__c, 200)` in all cases

Reply:
I would need to keep checking for this undefined values in the other cases because I need to "toString()" them.